TAO/tao/Storable_Base.h

```diff
@@ -112,6 +112,33 @@ namespace TAO
     virtual Storable_Base& operator << (ACE_INT64 ) = 0;
     virtual Storable_Base& operator >> (ACE_INT64 &) = 0;
 
+#if defined (ACE_HAS_CPP11)
+#include <type_traits>
+
```

Comment:
Include shouldn't be in the middle of the file

Reply:
Will move it to the top.